style.css

```diff
@@ -342,3 +346,122 @@ nav .status-bar {
   font-size: 22px;
   line-height: 43px;
 }
+
+.contact-section {
+  background: white;
+}
```

Comment:
You did an amazing job implementing the other requirements that previous reviewer requested you to do but kindly use flexbox in contact section as it is part of the the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements). You can use this [link](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for guidance how to use flexbox. Keep rocking 💪

Reply:
I used flexbox in the footer link ul